### ISSUE TITLE
Login screen accessibility issues

### ISSUE THREAD

**Issue by michellexliu** [autolab/Autolab#1373]
1. **Missing form labels:** email address and password fields do not have associated text labels, so the function/purpose of those fields may not be presented to screen reader users. To fix this, we need to either add a `<label>` element or use the `aria-label` attribute.
2. **Language missing:** The language of the document should be identified using the `lang` attribute (e.g., <html lang="en">) for screen readers to read the content in the appropriate language.
3. **Contrast errors:** The current blue color for links ("Developer Login", "Register", "Forgot your password", etc.) does not have a sufficient 4.5:1 contrast ratio, according to AA-accessibility guidelines.
